src/Middleware/ResponseCaching/src/ResponseCachingMiddleware.cs

```diff
@@ -29,6 +32,13 @@ public class ResponseCachingMiddleware
         private readonly IResponseCache _cache;
         private readonly IResponseCachingKeyProvider _keyProvider;
 
+        /// <summary>
+        /// Creates a new <see cref="ResponseCachingMiddleware"/>.
+        /// </summary>
+        /// <param name="next">The <see cref="RequestDelegate"/> representing the next middleware in the pipeline.</param>
+        /// <param name="options">The <see cref="ResponseCachingOptions"/> representing the options for this middleware.
```

Comment:
```suggestion
        /// <param name="options">The options for this middleware.
```

I think it's OK to not use a `<see>` tag here. I know the others do, but I'm not 100% on the appropriate syntax to reference `IOptions<ResponseCachingOptions>` (given the generics) and the docs will have the full signature and will cross-ref all the parameters types automatically anyway.